### ISSUE TITLE
Fix S2068 TP/FP: Issue title

### ISSUE THREAD

**Issue by hendrik-buchwald-sonarsource** [SonarSource/sonar-dotnet#7982]
### Description

In [SONARXML-169](https://sonarsource.atlassian.net/browse/SONARXML-169) support for hardcoded clear-text credentials in `web.config` files was added. It is currently [not working](https://sonarsource.atlassian.net/browse/SONARXML-182) but as soon as it gets fixed there will be an overlap with the rule for sonar-dotnet, i.e. issues will be reported twice. So even though it is a true positive, it would be best if sonar-dotnet does not raise an issue for this specific case anymore.

There is also a false positive that is raised for the attribute `passwordFormat` that sets the password format.

### Repro steps

```
<configuration>
  <system.web>
     <authentication mode="Forms">
      <forms>
        <credentials passwordFormat="Clear">  <!-- FP -->
          <user name="admin" password="password" />  <!-- unwanted TP -->
        </credentials>
      </forms>
    </authentication>
  </system.web>
</configuration>
```

### Expected behavior

No issues should be raised in `web.config` files for children of  `<configuration><system.web><authentication mode="Forms"><forms>` .

### Actual behavior

Issues are raised for `<credentials passwordFormat="Clear">` and `<user name="admin" password="password" />`.

**Comment by pavel-mikula-sonarsource** [comment]
Internal ticket [NET-1691](https://sonarsource.atlassian.net/browse/NET-1691)

[NET-1691]: https://sonarsource.atlassian.net/browse/NET-1691?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
